spec/unit/type/gluster_peer_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
```

Comment:
That's what the examples I'm working from do. I'll have `spec/unit/provider` as well and maybe `spec/integration/<something>` if that makes sense.